utilities/pipelines/platform/Set-AvmGitHubIssueOwnerConfig.ps1

```diff
@@ -95,6 +99,16 @@ function Set-AvmGitHubIssueOwnerConfig {
             if ($PSCmdlet.ShouldProcess(("owner [{0}] to issue [$($issue.title)]" -f $module.PrimaryModuleOwnerGHHandle), 'Assign')) {
                 # assign owner
                 $assign = gh issue edit $issue.url --add-assignee $module.PrimaryModuleOwnerGHHandle --repo $Repo
+
+                #assign owner team members
+                $ownerTeamMembers | ForEach-Object {
+                    gh issue edit $issue.url --add-assignee $_ --repo $Repo
+                }
```

Comment:
I guess with that, line 101 becomes redundant? Or at least I'd hope that the account behind `$module.PrimaryModuleOwnerGHHandle` is a member of the owner team. Thoughts?

Reply:
@AlexanderSehr the owner should also be part of the team, but I thought I also cater for the event, that he/she is not in the team for whatever reason.